src/commands/createTransactionCastVotes.js

```diff
@@ -49,26 +49,38 @@ export const actionCreator = vorpal => async ({ options }) => {
 	const {
 		passphrase: passphraseSource,
 		'second-passphrase': secondPassphraseSource,
-		vote,
-		unvote,
+		votes,
+		unvotes,
 	} = options;
 
-	if (!vote && !unvote) {
+	if (!votes && !unvotes) {
 		throw new Error('At least one of votes and/or unvotes options must be provided.');
```

Comment:
Should these be validation errors?

Reply:
No. There is no validation happening, just a check for existence. But we could come up with error messages too. Such as missingOptionError or so...

Reply:
Yeah, I just noticed the message wasn't red in table output...